make/modules/jdk.incubator.vector/Lib.gmk

```diff
@@ -37,3 +37,20 @@ ifeq ($(call isTargetOs, linux windows)+$(call isTargetCpu, x86_64)+$(INCLUDE_CO
 
   TARGETS += $(BUILD_LIBJSVML)
 endif
+
+ifeq ($(call isTargetOs, linux)+$(call isTargetCpu, riscv64)+$(INCLUDE_COMPILER2), true+true+true)
+  $(eval $(call SetupJdkLibrary, BUILD_LIBSLEEF, \
+      NAME := sleef, \
+      OPTIMIZATION := HIGH, \
+      SRC := libsleef/lib, \
+      EXTRA_SRC := libsleef/generated, \
+      DISABLED_WARNINGS_gcc := unused-function sign-compare tautological-compare ignored-qualifiers, \
+      DISABLED_WARNINGS_clang := unused-function sign-compare tautological-compare ignored-qualifiers, \
+      CFLAGS := $(CFLAGS_JDKLIB) -march=rv64gcv, \
+      LDFLAGS := $(LDFLAGS_JDKLIB) \
```

Comment:
Sorry for the late comment, I've been out ill. The LDFLAGS line is no longer needed and should be removed.

Reply:
So is the `LIBS` line as well. And the CFLAGS line should be just:
```
       CFLAGS := -march=rv64gcv, \
```

Reply:
No worries, it's not late. Take care!

Reply:
Do you suggest something like below?
```
       EXTRA_SRC := libsleef/generated, \
       DISABLED_WARNINGS_gcc := unused-function sign-compare tautological-compare ignored-qualifiers, \
       DISABLED_WARNINGS_clang := unused-function sign-compare tautological-compare ignored-qualifiers, \
-      CFLAGS := $(CFLAGS_JDKLIB) -march=rv64gcv, \
-      LDFLAGS := $(LDFLAGS_JDKLIB) \
-          $(call SET_SHARED_LIBRARY_ORIGIN), \
-      LIBS := $(JDKLIB_LIBS) \
+      CFLAGS := -march=rv64gcv \
```